docs/src/content/docs/reference/helm-chart-config.mdx

```diff
@@ -22,144 +28,6 @@ The configuration for the Helm chart is provided as a YAML file. It has the foll
-            <td>
-                Banner message (as HTML) to display at the very top of the page
-            </td>
-        </tr>
-        <tr>
-            <td>`welcomeMessageHTML`</td>
-            <td>String</td>
-            <td></td>
-            <td>
-                A custom welcome message to be shown on the landing page
-            </td>
-        </tr>
-        <tr>
-            <td>`additionalHeadHTML`</td>
-            <td>String</td>
-            <td></td>
-            <td>
-                Additional HTML to inject into the `<head>` of pages
-            </td>
-        </tr>
-        <tr>
-            <td>`createTestAccounts`</td>
-            <td>Boolean</td>
-            <td></td>
-            <td>If true, creates the users _testuser_ and _superuser_ (the username and password are the same).</td>
-        </tr>
-        <tr>
-            <td>`robotsNoindexHeader`</td>
-            <td>Boolean</td>
-            <td></td>
-            <td>If true, adds a noindex header to prevent search engine indexing</td>
-        </tr>
-        <tr>
-            <td>`seqSets.enabled`</td>
-            <td>Boolean</td>
-            <td></td>
-            <td>Enable/disable SeqSets. If false, `seqSets.crossRef` can be omitted.</td>
-        </tr>
-        <tr>
-            <td>`seqSets.crossRef`</td>
-            <td>Object</td>
-            <td></td>
-            <td>Configuration for CrossRef integration. Set to `null` to disable CrossRef integration (you can still use SeqSets without CrossRef DOIs).</td>
-        </tr>
-        <tr>
-            <td>`seqSets.crossRef.DOIPrefix`</td>
-            <td>String</td>
-            <td></td>
-            <td>The DOI prefix for SeqSets</td>
-        </tr>
-        <tr>
-            <td>`seqSets.crossRef.endpoint`</td>
-            <td>String</td>
-            <td></td>
-            <td>The API endpoint for CrossRef</td>
-        </tr>
-        <tr>
-            <td>`seqSets.crossRef.databaseName`</td>
-            <td>String</td>
-            <td></td>
-            <td>The database name for CrossRef</td>
-        </tr>
-        <tr>
-            <td>`seqSets.crossRef.email`</td>
-            <td>String</td>
-            <td></td>
-            <td>The email address associated with the CrossRef account</td>
-        </tr>
-        <tr>
-            <td>`seqSets.crossRef.organization`</td>
-            <td>String</td>
-            <td></td>
-            <td>The organization name for CrossRef</td>
-        </tr>
-        <tr>
-            <td>`seqSets.crossRef.hostUrl`</td>
-            <td>String</td>
-            <td></td>
-            <td>The host URL for CrossRef callbacks</td>
-        </tr>
-        <tr>
-            <td>`gitHubMainUrl`</td>
-            <td>String</td>
-            <td>"https://github.com/loculus-project/loculus"</td>
-            <td>The link that the GitHub icon in the footer points to</td>
-        </tr>
         <tr>
             <td>`dataUseTerms.enabled`</td>
```

Comment:
Why is dataUseTerms treated differently and not rendered through the schema?

Reply:
That was a copy-paste leftover, whoops! it was also in the schema already (but without the doc strings) - I've fixed it here now, thanks!: https://github.com/loculus-project/loculus/pull/3588/commits/b4a6ce4881531763f429f8e6b13b0afaba6d3203

Reply:
@fhennig, the datauseterms are not mentioned in the schema docs at all anymore somehow?

Reply:
oh yes damn, https://github.com/loculus-project/loculus/pull/3588/commits/13fd107ed5b2bb063a09266ef029541ae3f49644